Userland/Libraries/LibAudio/Sample.h

```diff
@@ -90,18 +90,22 @@ struct Sample {
         return new_frame;
     }
 
-    ALWAYS_INLINE Sample& log_pan(double const pan)
+    // Constant power panning
+    ALWAYS_INLINE Sample& pan(double const position)
     {
-        left *= linear_to_log(min(pan * -1 + 1.0, 1.0));
-        right *= linear_to_log(min(pan + 1.0, 1.0));
+        double const pi_over_2 = AK::Pi<double> * 0.5;
```

Comment:
Shouldn't these be `constexpr`? Is there something stopping that from happening?

Reply:
Yes, AK::Pi and the math functions are not constexpr in clang's mind. @Granddave 's original version had these as constexpr as well.